commit 326d53c8378984c50f29bd124d3f2b4a1242306c
Author: mjw <mjw@a5019735-40e9-0310-863c-91ae7b9d1cf9>
Date:   Fri Jun 16 09:33:35 2017 +0000

    ppc64 doesn't compile test_isa_2_06_partx.c without VSX support
    
    The #ifdef HAS_VSX guard is wrongly placed. It makes the standard
    include headers not be used. Causing a build failure. Fix by moving
    the #ifdef HAS_VSX after the standard includes.
    
    https://bugs.kde.org/show_bug.cgi?id=381272
    
    git-svn-id: svn://svn.valgrind.org/valgrind/trunk@16450 a5019735-40e9-0310-863c-91ae7b9d1cf9

diff --git a/none/tests/ppc32/test_isa_2_06_part1.c b/none/tests/ppc32/test_isa_2_06_part1.c
index 7cd4930..7a14c6d 100644
--- a/none/tests/ppc32/test_isa_2_06_part1.c
+++ b/none/tests/ppc32/test_isa_2_06_part1.c
@@ -20,13 +20,14 @@
  The GNU General Public License is contained in the file COPYING.
  */
 
-#ifdef HAS_VSX
-
 #include <stdio.h>
 #include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
 #include <malloc.h>
+
+#ifdef HAS_VSX
+
 #include <altivec.h>
 
 #ifndef __powerpc64__
diff --git a/none/tests/ppc32/test_isa_2_06_part2.c b/none/tests/ppc32/test_isa_2_06_part2.c
index c7bf4fe..2ee7b53 100644
--- a/none/tests/ppc32/test_isa_2_06_part2.c
+++ b/none/tests/ppc32/test_isa_2_06_part2.c
@@ -20,17 +20,18 @@
  The GNU General Public License is contained in the file COPYING.
  */
 
-#ifdef HAS_VSX
-
 #include <stdio.h>
 #include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
 #include <malloc.h>
-#include <altivec.h>
 #include <math.h>
 #include <unistd.h>    // getopt
 
+#ifdef HAS_VSX
+
+#include <altivec.h>
+
 #ifndef __powerpc64__
 typedef uint32_t HWord_t;
 #else
diff --git a/none/tests/ppc32/test_isa_2_06_part3.c b/none/tests/ppc32/test_isa_2_06_part3.c
index 8c74c09..5ebc1a5 100644
--- a/none/tests/ppc32/test_isa_2_06_part3.c
+++ b/none/tests/ppc32/test_isa_2_06_part3.c
@@ -20,17 +20,18 @@
  The GNU General Public License is contained in the file COPYING.
  */
 
-#ifdef HAS_VSX
-
 #include <stdio.h>
 #include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
 #include <malloc.h>
-#include <altivec.h>
 #include <math.h>
 #include <unistd.h>    // getopt
 
+#ifdef HAS_VSX
+
+#include <altivec.h>
+
 #ifndef __powerpc64__
 typedef uint32_t HWord_t;
 #else

diff --git a/none/tests/ppc64/test_isa_2_06_part1.c b/none/tests/ppc64/test_isa_2_06_part1.c
index 7cd4930..7a14c6d 100644
--- a/none/tests/ppc64/test_isa_2_06_part1.c
+++ b/none/tests/ppc64/test_isa_2_06_part1.c
@@ -20,13 +20,14 @@
  The GNU General Public License is contained in the file COPYING.
  */
 
-#ifdef HAS_VSX
-
 #include <stdio.h>
 #include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
 #include <malloc.h>
+
+#ifdef HAS_VSX
+
 #include <altivec.h>
 
 #ifndef __powerpc64__
diff --git a/none/tests/ppc64/test_isa_2_06_part2.c b/none/tests/ppc64/test_isa_2_06_part2.c
index c7bf4fe..2ee7b53 100644
--- a/none/tests/ppc64/test_isa_2_06_part2.c
+++ b/none/tests/ppc64/test_isa_2_06_part2.c
@@ -20,17 +20,18 @@
  The GNU General Public License is contained in the file COPYING.
  */
 
-#ifdef HAS_VSX
-
 #include <stdio.h>
 #include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
 #include <malloc.h>
-#include <altivec.h>
 #include <math.h>
 #include <unistd.h>    // getopt
 
+#ifdef HAS_VSX
+
+#include <altivec.h>
+
 #ifndef __powerpc64__
 typedef uint32_t HWord_t;
 #else
diff --git a/none/tests/ppc64/test_isa_2_06_part3.c b/none/tests/ppc64/test_isa_2_06_part3.c
index 8c74c09..5ebc1a5 100644
--- a/none/tests/ppc64/test_isa_2_06_part3.c
+++ b/none/tests/ppc64/test_isa_2_06_part3.c
@@ -20,17 +20,18 @@
  The GNU General Public License is contained in the file COPYING.
  */
 
-#ifdef HAS_VSX
-
 #include <stdio.h>
 #include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
 #include <malloc.h>
-#include <altivec.h>
 #include <math.h>
 #include <unistd.h>    // getopt
 
+#ifdef HAS_VSX
+
+#include <altivec.h>
+
 #ifndef __powerpc64__
 typedef uint32_t HWord_t;
 #else
